### PR TITLE
Set up i18n

### PR DESCRIPTION
**Pull request by PatrikNguyen**
## Task TicketID: 40906
- [#40906](https://edu-redmine.sun-asterisk.vn/issues/40906)

## WHAT (optional)
- Set up i18n and a small demo for test on homepage

## Evidence (Screenshot or Video)
Screenshot from 2021-08-25 09-36-17

### CODE REVIEW

**Review by cuongnk-2655 — APPROVED** [review]

**Comment by phatntt1999** [comment]
ready

**Comment by phatntt1999** [comment]
ready

**Comment by PatrikNguyen** [comment]
ready

File: app/helpers/pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```
Comment (ghost) [inline app/helpers/pages_helper.rb]: Chưa dùng Helper này xóa đi em

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1><%= t ".title" %></h1>
+<p><%= t ".greeting" %></p>
+<%= link_to t(".eng"), locale:"en" %>
```
Comment (ghost) [inline app/views/pages/home.html.erb]:
```suggestion
<%= link_to t(".eng"), locale: "en" %>
```

**Comment by PatrikNguyen** [comment]
ready

**Review by ghost — APPROVED** [review]

**Pull request merged**